src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.TimeZone/Classes/OneTimeZone.cs

```diff
@@ -0,0 +1,96 @@
+﻿// Copyright (c) Microsoft Corporation
+// The Microsoft Corporation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text.Json;
+
+namespace Microsoft.PowerToys.Run.Plugin.TimeZone.Classes
+{
+    /// <summary>
+    /// A time zone
+    /// </summary>
+    public sealed class OneTimeZone
```

Comment:
Any particular reason for `OneTimeZone` ? Why not just `TimeZone` ? :)

Reply:
It's impossible to rename it "TimeZone" because, the namespace contain it too
`Microsoft.PowerToys.Run.Plugin.TimeZone`

Reply:
We could use `TimeZoneProperties`. That sounds better